aws/requests.yaml

```diff
@@ -4,6 +4,10 @@ releases:
         - name: coredns
           version: ">= 1.4.0"
           issue: https://github.com/giantswarm/giantswarm/issues/13771
+    - name: ">= 13.1.0"
```

Comment:
But why not `>13.0.0`? so it includes, say, 13.0.1?

Reply:
wrong component

Reply:
> just wondering, why not >13.0.0? so it includes, say, 13.0.1?

I didn't think of that. The ranges are a bit tricky because 14.0.0 already exists but this is done now and CI is passing.

Reply:
oki doki!